tests/bluetooth/tester/overlay-bt_ll_sw_split.conf

```diff
@@ -38,6 +38,11 @@ CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=3
 CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
 CONFIG_BT_ISO_MAX_BIG=2
 
+# Match supported periodic advertising ext adv instances
+CONFIG_BT_CTLR_ADV_SYNC_SET=2
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
```

Comment:
Why do we need advanced features?

Reply:
> Some of the Kconfigs are under advanced features from the early days, have not gotten to moving them out.

Alright, so they should not really be there? 
Would be nice to add a GH issue on this that we can refer to, or at least have this in the comments for why we are enabling advanced features here